### PR TITLE
Allow specifying start/end for 'show()', thus speed up display (Issue #1)

### PR DESCRIPTION
**Pull request by mungewell**
Pull request for Issue #1

### CODE REVIEW

**Comment by samveen** [comment]
@mungewell For completeness, would you please add a section in the README with an example of how the update function is to be used?

File: PicoOled13.py [hunk 1]
```diff
@@ -138,9 +138,17 @@ def init_display(self):
 
         self.on()
 
-    def show(self):
+    def show(self, start=0, end=-1):
+        if end < 0:
+            end = self.height
+
+        if end < start:
+            temp = end
```
Comment (samveen) [inline PicoOled13.py:146]: For swapping, how about this:

start, end = end, start

**Comment by mungewell** [comment]
Will add to 'readme', but I am away for a few days.

As-is you can call as follows.
`show()` - refresh whole screen
`show(32)` - refresh from line 32 to bottom of screen
`show(32,48)` - refresh between lines 32 and 48

**Comment by federicoha** [comment]
Is my understanding (if not please forget this msg) allow to set the size of the font, how that can be? if code sample added, will be appreciated.

**Comment by samveen** [comment]
@federicoha the display class code maintains the contents of the screen as an image internally, modifies that image, and then writes the image to the device when `show()` is called.  This MR allows the ability to redraw only a part of the screen, instead of redrawing the whole screen.

 In this implementation, the font is fixed size unfortunately. The ability to resize fonts requires more memory  (and programming ability ) than is available on the raspberry pi pico (and me), which was the primary development environment and use for this display and code.

If you can give me an idea of what you are looking for, I can take a look at implementing resizable fonts if it is possible, under a new issue.

**Comment by federicoha** [comment]
thanks for the quick reply @samveen 
currently, the OLED 1.3 shows characters with an 8 size (small one), would be great to have a way to instruct the display text with a bigger font size,  If you only show one line of text, the display looks big compared with the text to display
Something like this is what i think, but you know best how can be

`display.text("text",0,0,0xffff, **size=x**)`

allow to show the text in a different font size (not sure how big can be) and also an optional feature like centering the text, but could be the 2.0 version :)

thanks in advance

**Pull request merged**